src/sentry/static/sentry/app/views/organizationDashboard/index.jsx

```diff
@@ -13,51 +14,58 @@ import getProjectsByTeams from 'app/utils/getProjectsByTeams';
 import withTeams from 'app/utils/withTeams';
 import withProjects from 'app/utils/withProjects';
 
-class Dashboard extends React.Component {
+class Dashboard extends AsyncComponent {
   static propTypes = {
     teams: PropTypes.array,
     projects: PropTypes.array,
   };
 
   componentWillMount() {
     $(document.body).addClass('org-dashboard');
+    super.componentWillMount();
   }
   componentWillUnmount() {
     $(document.body).removeClass('org-dashboard');
+    super.componentWillUnmount();
   }
 
-  render() {
-    const {projects, teams} = this.props;
+  getEndpoints() {
+    const {orgId} = this.props.params;
+    return [['projectsWithStats', `/organizations/${orgId}/projects/?statsPeriod=24h`]];
+  }
+
+  renderBody() {
+    const {teams, projects, params} = this.props;
     const {projectsByTeam} = getProjectsByTeams(teams, projects);
     const projectKeys = Object.keys(projectsByTeam);
 
+    const {projectsWithStats} = this.state;
+    const getStats = id => projectsWithStats.find(project => id === project.id).stats;
+
     return (
       <div>
-        <ProjectNav />
-        <div>
-          {projectKeys.map((slug, index) => {
-            return (
-              <TeamSection key={slug} showBorder={index !== projectKeys.length - 1}>
-                <TeamTitleBar justify="space-between" align="center">
-                  <TeamName>{`#${slug}`}</TeamName>
-                  <TeamMembers teamId={slug} orgId={this.props.params.orgId} />
-                </TeamTitleBar>
-                <ProjectCards>
-                  {projectsByTeam[slug].map(project => {
-                    return (
-                      <ProjectCardWrapper
-                        key={project.id}
-                        width={['100%', '50%', '33%', '25%']}
-                      >
-                        <ProjectCard project={project} />
-                      </ProjectCardWrapper>
-                    );
-                  })}
-                </ProjectCards>
-              </TeamSection>
-            );
-          })}
-        </div>
+        {projectKeys.map((slug, index) => {
+          return (
+            <TeamSection key={slug} showBorder={index !== projectKeys.length - 1}>
```

Comment:
imo, an intermediate variable for `showBorder` would make it a bit easier to read